api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/WorksQuery.scala

```diff
@@ -17,6 +17,10 @@ case object WorksMultiMatcher {
   def apply(q: String): BoolQuery = {
     boolQuery()
       .should(
+        // we're running elasticsearch 7.9 at the moment - upgrading to 7.11 will give us the ability to run case
+        // insensitive prefixQueries, which will solve our "old and new london" problem
+        // (see https://github.com/wellcomecollection/catalogue/issues/1336)
+        prefixQuery("data.title.keyword", q).boost(1000),
```

Comment:
@harrisonpim - we're on 7.11 now - do you want to add this to this PR?

Reply:
yep

Reply:
To keep a record of our conversation, elastic4s 7.10 doesn't have this added.